Add test case IDs to automate report of results

pbrilla-rh opened the pull request:
Adding ID for each test case, so it is linked together during reporting increasing tracebility

mwperina commented on tests/README.md:
```diff
@@ -155,6 +155,15 @@ exit_code, output = ctrl.run_python("python/monitor.py")
 A full example of how to use the python bindings can be found in the [monitor open-close test](./tests/tier0/monitor-open-close/).
 
+### Generating test ID
+
```
It would nice to add something like:

Each test should be clearly identified by unique ID, so when adding a new test, please run following command to assign ID to the new test:

mwperina commented on tests/README.md:
````diff
@@ -155,6 +155,15 @@ exit_code, output = ctrl.run_python("python/monitor.py")
+tmt test id .
+New id 'UUID' added to test '/tests/path_to_your_new_test'.
+...
+```
````
Hmm, shouldn't the formatting look like below?

```shell
cd ~/bluechi/tests
tmt test id .
```

New id 'UUID' is added to the new test.

pbrilla-rh replied:
New id 'UUID' is added to the new test. 
This is return in CLI from tmt command

mwperina replied:
Ah, OK, now it makes sense to me

mwperina reviewed (changes requested):
Please squash the commits into a single one and add some nice description about the change also to the commit message

pbrilla-rh committed:
Add test case IDs to increase tracebility

Each test case should have unique UUID to help with traceability of test runs.
With UUID for each test case, it is easier to follow trends in stability over time.

Signed-off-by: Pavol Brilla <pbrilla@redhat.com>

mwperina reviewed (approved):
LGTM

engelmi commented on tests/README.md:
````diff
@@ -155,6 +155,17 @@ exit_code, output = ctrl.run_python("python/monitor.py")
+
+```shell
+cd ~/bluechi/tests
+tmt test id .
````
```suggestion
$ cd ~/bluechi/tests
$ tmt test id .
```
NIT: To distinguish between command and output.

engelmi reviewed (approved):
LGTM
Only a small NIT. 
Would probably be good to add a check for the ID to the CI so that newly added tests really have an ID assigned. But that can be done in a separate PR.

pbrilla-rh committed:
Update tests/README.md

thanks this makes sense

Co-authored-by: Michael Engel <engel.michael@hotmail.de>
Signed-off-by: Pavol Brilla <48551923+pbrilla-rh@users.noreply.github.com>
